awx/settings/defaults.py

```diff
@@ -1140,12 +1140,12 @@ def IS_TESTING(argv=None):
         'awx.main': {
             'handlers': ['null']
         },
-        'awx.main.commands.run_callback_receiver': {
-            'handlers': ['callback_receiver'],
-        },
         'awx.main.dispatch': {
             'handlers': ['dispatcher'],
         },
+        'awx.main.dispatch.worker.callback': {
+            'level': 'INFO'  # in debug mode, includes full callback data
```

Comment:
Don't you have to specify at least one `handler`?

Reply:
no, it propagates up through the parent handlers. In the prior version of this (which I may now move back towards), I think the job event payload was getting logged twice, because both the dispatcher and the awx logger were going to the console. Because of that, I was quite notably removing the dispatch handler in the development settings. That might have change anyway for unrelated reasons since I started on this.